spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractBinder.java

```diff
@@ -128,7 +136,10 @@ protected void onInit() throws Exception {
 		if (StringUtils.isEmpty(group)) {
 			Assert.isTrue(!properties.isPartitioned(), "A consumer group is required for a partitioned subscription");
 		}
-		return doBindConsumer(name, group, target, properties);
+		Binding<T> binding = doBindConsumer(name, group, target, properties);
+		registerErrorInfrastructure(binding,group,properties);
```

Comment:
I know, that's why I am concerned. 
You see you don't have whitespaces after commas.
That looks a bit clumsy and hard to review, believe me...